Dockerfile

```diff
@@ -0,0 +1,53 @@
+#  Copyright (C) 2024 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# NAV2 extensions build file
+# Use ROS Humble as the base image
+FROM ros:humble-ros-base
+
+ARG GIT_BRANCH="develop"
+ARG BUILD_DATE="NULL"
+ARG VERSION="NULL"
+ARG VCS_REF="NULL"
+ENV ROS_DISTRO=humble
+
+RUN mkdir -p /root/c1t_ws/src/navigation2_extensions
+
+# Set the working directory
+WORKDIR /root/c1t_ws/src/
+
+RUN git clone https://github.com/usdot-fhwa-stol/carma-msgs && \
```

Comment:
Would it be better to put these into install_depepdencies.sh?

Reply:
Updated